examples/abitest/grpc/src/lib.rs

```diff
@@ -0,0 +1,19 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+pub mod proto {
+    tonic::include_proto!("oak.examples.abitest");
+}
```

Comment:
Basically because of #1093: the Tonic-native gRPC service code can't (yet) co-exist with the Oak-specific gRPC service code (both code generators expect to output `OUTDIR/oak.examples.abitest.rs`).  

(I'm hoping that we can fix #1093 in such a way that all the different proto-generated code can live together in `<example>::proto`, and then these `<example>_grpc/` directories should evaporate.)